src/interfaces/element-props.ts

```diff
@@ -1,9 +1,32 @@
+export const Phases = {
+  Gas: "Gas",
+  Liquid: "Liquid",
+  Solid: "Solid"
+} as const;
+
+export const Categories = {
+  DiatomicNonmetal: "diatomic nonmetal",
+  NobleGas: "noble gas",
+  AlkaliMetal: "alkali metal",
+  AlkalineEarthMetal: "alkaline earth metal",
+  Metalloid: "metalloid",
+  PolyatomicNonmetal: "polyatomic nonmetal",
+  PostTransitionMetal: "post-transition metal",
+  TransitionMetal: "transition metal",
+  Lanthanide: "lanthanide",
+  Actinide: "actinide"
+} as const;
+
+export type Category = typeof Categories[keyof typeof Categories];
+
+export type Phase = typeof Phases[keyof typeof Phases];
+
 export interface ElementProps {
   name: string;
   appearance?: string;
   atomicMass: number;
   boil?: number;
-  category: string;
+  category: Category | string;
   color?: string;
   /**
    * Given in `g/l` for gases and `g/cm³` for solids and liquids
```

Comment:
Sir @maneetgoyal  this is not a part if this PR but should we have a `JSdoc` style comment in b/w the interface?

Reply:
Yes, I added it. Comments like these can be ingested by TypeDoc, so it's fine for auto-doc generation purposes.